src/components/select/multi-select/multi-select.test.js

```diff
@@ -0,0 +1,1281 @@
+        <td>Poe</td>
+        <td>Accountant</td>
+      </OptionRow>
+      <OptionRow value="4" text="Jill Moe">
+        <td>Jill</td>
+        <td>Moe</td>
+        <td>Engineer</td>
+      </OptionRow>
+      <OptionRow value="5" text="Bill Zoe">
+        <td>Bill</td>
+        <td>Zoe</td>
+        <td>Astronaut</td>
+      </OptionRow>
+    </MultiSelect>
+  );
+};
+
+const MultiSelectMaxOptionsComponent = ({ ...props }) => {
+  const maxSelectionsAllowed = 2;
+  const [selectedPills, setSelectedPills] = React.useState([]);
+
+  function onChangeHandler(event) {
+    if (event.target.value.length <= maxSelectionsAllowed) {
+      setSelectedPills(event.target.value);
+    }
+  }
+
+  return (
+    <MultiSelect
+      value={selectedPills}
+      onChange={onChangeHandler}
+      disablePortal
+      openOnFocus
+      label="color"
+      {...props}
+    >
+      <Option text="Amber" value="1" />
+      <Option text="Black" value="2" />
+      <Option text="Blue" value="3" />
+      <Option text="Brown" value="4" />
+      <Option text="Green" value="5" />
+      <Option text="Orange" value="6" />
+      <Option text="Pink" value="7" />
+      <Option text="Purple" value="8" />
+      <Option text="Red" value="9" />
+      <Option text="White" value="10" />
+      <Option text="Yellow" value="11" />
+    </MultiSelect>
+  );
+};
+
+const MultiSelectOnFilterChangeEventComponent = ({ onChange, ...props }) => {
+  const [state, setState] = React.useState("");
+
+  const setValue = ({ target }) => {
+    setState(target.value.rawValue);
+    if (onChange) {
+      onChange(target);
+    }
+  };
+
+  return (
+    <MultiSelect
+      label="color"
+      value={state}
+      labelInline
+      onChange={setValue}
+      {...props}
+    >
+      <Option text="Amber" value="1" />
+      <Option text="Black" value="2" />
+      <Option text="Blue" value="3" />
+      <Option text="Brown" value="4" />
+      <Option text="Green" value="5" />
+    </MultiSelect>
+  );
+};
+
+const MultiSelectCustomColorComponent = ({ ...props }) => {
+  return (
+    <MultiSelect label="color" labelInline defaultValue={["1", "3"]} {...props}>
+      <Option text="Amber" value="1" borderColor="#FFBF00" fill />
+      <Option text="Black" value="2" borderColor="blackOpacity65" fill />
+      <Option text="Blue" value="3" borderColor="productBlue" />
+      <Option text="Brown" value="4" borderColor="brown" fill />
+      <Option text="Green" value="5" borderColor="productGreen" />
+      <Option text="Orange" value="6" borderColor="orange" />
+      <Option text="Pink" value="7" borderColor="pink" />
+      <Option text="Purple" value="8" borderColor="purple" />
+      <Option text="Red" value="9" borderColor="red" fill />
+      <Option text="White" value="10" borderColor="white" />
+      <Option text="Yellow" value="11" borderColor="yellow" fill />
+    </MultiSelect>
+  );
+};
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
```

Comment:
we will use the  import of those values from our `constants` (are added be @DianeGit22 )